workers/api-server/src/http/external_match.rs

```diff
@@ -26,6 +30,27 @@ use crate::{
 /// The error message returned when the relayer fails to process an external
 /// match request
 const ERR_FAILED_TO_PROCESS_EXTERNAL_MATCH: &str = "failed to process external match request";
+/// The message returned when no atomic match is found
+const NO_ATOMIC_MATCH_FOUND: &str = "no atomic match found";
+
+// -----------
+// | Helpers |
+// -----------
+
+/// Await a response from the external matching engine
+///
+/// TODO: Add a timeout
```

Comment:
do we actually need a timeout here? the external matching engine definitely resolves with either `AtomicMatchFound` or `NoAtomicMatchFound`, so it's not as though we use a timeout to infer an error or absence of a match.

if the matching process needs to fit within some time constraint, API clients can set their own timeouts

Reply:
I guess, I can understand wanting a timeout to obviate issues with the system bus, rather than the matching engine... in which case let's be sure to set it relatively high, as we have no reason to expect any issues here